### ISSUE TITLE
Access token and using the same account throughout

### ISSUE THREAD

**Issue by marxo** [dropbox/dropbox-js#214]
I want to create a static web app that would always use my account and files in a particular folder. I'm using dropbox-js for utilizing the API. I think i should use the access_token that I generate from the App Console, but when I try this:

    var client = new Dropbox.Client({
        key: "y2q6wsnp17gvbcg",
        token: "my-access-token-here",
    });
I get nothing. Basically, I use dropbox.readdir("pathtofolder") and without the token it works, obviously only when I'm authenticated, but with the access token it just does nothing.

Anyone had any experience with this?

**Comment by pwnall** [comment]
Can you look at the network tab and see what happens?

**Comment by marxo** [comment]
All 200 OK. I host the index.html inside my Dropbox, and the `isAuthenticated()` returns `true`.

**Comment by marxo** [comment]
BTW I've also ran `curl https://api.dropbox.com/1/metadata/auto -H "Authorization:Be
arer my-token-here"` from Terminal with success.

**Comment by smarx** [comment]
See also on Stack Overflow: http://stackoverflow.com/questions/28068505/dropbox-api-always-access-my-account-dropbox-js

**Comment by pwnall** [comment]
@marxo Do you see the `/metadata` network request for the `readdir` call in the Network tab? If so, it seems like either you have a bug, or dropbox.js chokes on something in the output. Is there any chance you can point me to the code that you're working on, so I can see what's broken?

**Comment by marxo** [comment]
@pwnall Nope, no /metadata call in the Network tab. I'll check if I can disclose the snippet and get back to you if it's ok.

**Comment by DavidR91** [comment]
I am seeing the same behaviour when specifying the key and a pre-generated access token

I'm assuming this is an auth bug somewhere 

Unfortunately I' can't contribute any useful network trace because the machine that is experiencing this is a console-only server

**Comment by DavidR91** [comment]
Looking [here](https://github.com/dropbox/dropbox-js/blob/stable/src/client.coffee#L16), the problem seems to be that whilst the constructor comments state it can take options.token... it doesn't appear to actually make use of it?

**Comment by pwnall** [comment]
https://github.com/dropbox/dropbox-js/blob/stable/src/client.coffee#L36
